code/modules/surgery/advanced/brainwashing.dm

```diff
@@ -54,7 +54,7 @@
 	if(!target.mind)
 		to_chat(user, span_warning("[target] doesn't respond to the brainwashing, as if [target.p_they()] lacked a mind..."))
 		return FALSE
-	if(HAS_TRAIT(target, TRAIT_MINDSHIELD))
+	if(HAS_MIND_TRAIT(target, TRAIT_UNCONVERTABLE))
```

Comment:
check for both trait mindshield and trait uncovertable here, there's no reason to change it according to the intent of this PR (which is to fix this for abductors).